src/measure_theory/measure/null_measurable.lean

```diff
@@ -280,6 +280,10 @@ lemma measure_union₀' (hs : null_measurable_set s μ) (hd : ae_disjoint μ s t
   μ (s ∪ t) = μ s + μ t :=
 by rw [union_comm, measure_union₀ hs hd.symm, add_comm]
 
+lemma measure_add_measure_compl₀ {s : set α} (hs : null_measurable_set s μ) :
+  μ s + μ sᶜ = μ univ :=
+by rw [← measure_union₀' hs(@disjoint_compl_right _ _ s).ae_disjoint, union_compl_self]
```

Comment:
```suggestion
by rw [← measure_union₀' hs ae_disjoint_compl_right, union_compl_self]
```
Also, please use this lemma to prove `measure_add_measure_compl`.

Reply:
Thanks, done.